browser/android/brave_perf_predictor_tab_helper_delegate_android_impl.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_ANDROID_BRAVE_PERF_PREDICTOR_TAB_HELPER_DELEGATE_ANDROID_IMPL_H_
+#define BRAVE_BROWSER_ANDROID_BRAVE_PERF_PREDICTOR_TAB_HELPER_DELEGATE_ANDROID_IMPL_H_
+
+#include "brave/components/brave_perf_predictor/browser/perf_predictor_tab_helper_delegate_android.h"
+
+namespace brave {
+namespace android {
+
+class BravePerfPredictorTabHelperDelegateAndroidImpl
```

Comment:
I don't understand the point of this class. Why don't you just pass a repeating callback to PerfPredictorTabHelper? Also we don't use `android` as a namespace. Not sure who did `chrome::android::BraveShieldsContentSettings`, but that is definitely not following the standard conventions

Reply:
actually the best fix is to just get rid of `DispatchSavedBandwidth` altogether and add a pref observer for `kBandwidthSavedBytes` instead either in BraveShieldsContentSettings or in java